apps/src/gamelab/AnimationPicker/index.jsx

```diff
@@ -1,15 +1,76 @@
+var _ = require('../../lodash');
 var AnimationPickerBody = require('./AnimationPickerBody.jsx');
 var Dialog = require('../../templates/DialogComponent.jsx');
+var HiddenUploader = require('../../assetManagement/HiddenUploader.jsx');
+var styles = require('./styles');
+var utils = require('../../utils');
+
+/** @enum {string} */
+var View = {
+  PICKER: 'PICKER',
+  UPLOAD_IN_PROGRESS: 'UPLOAD_IN_PROGRESS',
+  ERROR: 'ERROR'
+};
```

Comment:
I wonder if we should build (or find) a simple util that deals with this pattern, so that I instead do something like:
```
var View = Enum(
  'PICKER',
  'UPLOAD_IN_PROGRESS',
  'ERROR'
);
```
Slightly more succinct, makes it more clear what it is, and reduces the risk fo unintentionally having a key/value be different. Not sure how much we care though. Any thoughts?